src/components/telemetry_monitor/test/CMakeLists.txt

```diff
@@ -74,4 +74,10 @@ endif()
 
 create_test("telemetry_monitor_test" "${SOURCES}" "${LIBRARIES}")
 
-file(COPY ${COMPONENTS_DIR}/telemetry_monitor/test/log4cxx.properties DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+if (${LOGGER_NAME} STREQUAL "LOG4CXX")
+  file(COPY ${COMPONENTS_DIR}/telemetry_monitor/test/log4cxx.properties DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+endif()
+
```

Comment:
can this be else?

Reply:
Changed in https://github.com/smartdevicelink/sdl_core/pull/3571/commits/31860450ad5362ae084e02152f5b14f47988ed7a